apps/web-giddh/src/assets/locale/sidebar-menu/en.json

```diff
@@ -163,7 +178,7 @@
                 },
                 {
                     "label": "Bill Settings",
-                    "link": "/pages/purchase-management/purchase/settings",
+                    "link": "/pages/vouchers/preview/purchase/settings",
```

Comment:
define separately for both voucher versions

Reply:
Done